dcat/rdf/dcat.ttl

```diff
@@ -239,1389 +584,1249 @@ dcat:CatalogRecord
+  skos:scopeNote "Use to characterize a relationship between datasets, and potentially other resources, where the nature of the relationship is known but is not adequately characterized by the standard Dublin Core properties (dct:hasPart, dct:isPartOf, dct:conformsTo, dct:isFormatOf, dct:hasFormat, dct:isVersionOf, dct:hasVersion, dct:replaces, dct:isReplacedBy, dct:references, dct:isReferencedBy, dct:requires, dct:isRequiredBy) or PROV-O properties (prov:wasDerivedFrom, prov:wasInfluencedBy, prov:wasQuotedFrom, prov:wasRevisionOf, prov:hadPrimarySource, prov:alternateOf, prov:specializationOf)"@en ;
+  skos:scopeNote "Viene utilizzato per caratterizzare la relazione tra insiemi di dati, e potenzialmente altri tipi di risorse, nei casi in cui la natura della relazione è nota ma non adeguatamente caratterizzata dalle proprietà dello standard 'Dublin Core' (dct:hasPart, dct:isPartOf, dct:conformsTo, dct:isFormatOf, dct:hasFormat, dct:isVersionOf, dct:hasVersion, dct:replaces, dct:isReplacedBy, dct:references, dct:isReferencedBy, dct:require, dct:isRequiredBy) o dalle propietà fornite da PROV-O  (prov:wasDerivedFrom, prov:wasInfluencedBy, prov:wasQuotedFrom, prov:wasRevisionOf, prov: hadPrimarySource, prov:alternateOf, prov:specializationOf)"@it ;
+.
+dcat:Resource
+  a owl:Class ;
+  rdfs:comment "Recurso publicado o curado por un agente único."@es ;
+  rdfs:comment "Resource published or curated by a single agent."@en ;
+  rdfs:comment "Risorsa pubblicata o curata da un singolo agente."@it ;
+  rdfs:comment "Zdroj publikovaný či řízený jediným činitelem"@cs ;
+  rdfs:label "Catalogued resource"@en ;
+  rdfs:label "Katalogizovaný zdroj"@cs ;
+  rdfs:label "Recurso catalogado"@es ;
+  rdfs:label "Risorsa catalogata"@it ;
+  rdfs:subClassOf [
+      a owl:Restriction ;
+      owl:minCardinality "1"^^xsd:nonNegativeInteger ;
+      owl:onProperty dct:type ;
+    ] ;
+  skos:changeNote "New class added in DCAT 2.0."@en ;
+  skos:changeNote "Nová třída přidaná ve verzi DCAT 2.0"@cs ;
+  skos:changeNote "Nueva clase agregada en DCAT 2.0."@es ;
+  skos:changeNote "Nuova classe aggiunta in DCAT 2.0."@it ;
+  skos:definition "Recurso publicado o curado por un agente único."@es ;
+  skos:definition "Resource published or curated by a single agent."@en ;
+  skos:definition "Risorsa pubblicata o curata da un singolo agente."@it ;
+  skos:definition "Zdroj publikovaný či řízený jediným činitelem"@cs ;
+  skos:editorialNote "Estado: esta clase se aceptó por el equipo de revisión de DCAT, pero los detalles de la definición y restricciones no están resueltos aún."@es ;
+  skos:editorialNote "Status: Class accepted by DCAT revision team, but details of definition and constraints not yet resolved."@en ;

```

Comment:
I think we have finalized the definition of dcat:Resource, so we can drop this editorial note and the related translations
```suggestion
```